src/testRunner/unittests/incrementalParser.ts

```diff
@@ -832,157 +832,162 @@ module m3 { }\
             insertCode(source, index, "Fo");
         });
 
-        describe("comment directives", () => {
-            const tsIgnoreComment = "// @ts-ignore";
-            const textWithIgnoreComment = `const x = 10;
-function foo() {
-    // @ts-ignore
-    let y: string = x;
-    return y;
-}
-function bar() {
-    // @ts-ignore
-    let z : string = x;
-    return z;
-}
-function bar3() {
-    // @ts-ignore
-    let z : string = x;
-    return z;
-}
-foo();
-bar();
-bar3();`;
-            verifyScenario("when deleting ts-ignore comment", verifyDelete);
-            verifyScenario("when inserting ts-ignore comment", verifyInsert);
-            verifyScenario("when changing ts-ignore comment to blah", verifyChangeToBlah);
-            verifyScenario("when changing blah comment to ts-ignore", verifyChangeBackToDirective);
-            verifyScenario("when deleting blah comment", verifyDeletingBlah);
-            verifyScenario("when changing text that adds another comment", verifyChangeDirectiveType);
-            verifyScenario("when changing text that keeps the comment but adds more nodes", verifyReuseChange);
-
-            function verifyCommentDirectives(oldText: IScriptSnapshot, newTextAndChange: { text: IScriptSnapshot; textChangeRange: TextChangeRange; }) {
-                const { incrementalNewTree, newTree } = compareTrees(oldText, newTextAndChange.text, newTextAndChange.textChangeRange, -1);
-                assert.deepEqual(incrementalNewTree.commentDirectives, newTree.commentDirectives);
-            }
-
-            function verifyScenario(scenario: string, verifyChange: (atIndex: number, singleIgnore?: true) => void) {
-                it(`${scenario} - 0`, () => {
-                    verifyChange(0);
-                });
-                it(`${scenario} - 1`, () => {
-                    verifyChange(1);
-                });
-                it(`${scenario} - 2`, () => {
-                    verifyChange(2);
-                });
-                it(`${scenario} - with single ts-ignore`, () => {
-                    verifyChange(0, /*singleIgnore*/ true);
-                });
-            }
-
-            function getIndexOfTsIgnoreComment(atIndex: number) {
-                let index: number;
-                for (let i = 0; i <= atIndex; i++) {
-                    index = textWithIgnoreComment.indexOf(tsIgnoreComment);
+        for (const tsIgnoreComment of [
+            "// @ts-ignore",
+            "/* @ts-ignore */",
+            "/*\n  @ts-ignore */"

```

Comment:
The one with `/*\n @ts-ignore */` wont test `verifyChangeToBlah` etc correctly right ? since it will not remove ts-ignore comment?

Reply:
Thanks for the catch. I think you're referring to the `"" /".length` lines later on? If so, great, changed. If not, I think I'm still missing something.